source/_posts/2025-04-16-state-of-the-open-home-recap.markdown

```
@@ -0,0 +1,152 @@
+  <li>Espressif has donated to the Foundation</li>
+  <li>Announced upcoming Works with Home Assistant partners</li>
+  <li>24 May will be Home Assistant Community Day with many meetups</li>
+  <li>Music Assistant and HACS are now Foundation projects</li>
+  <li>The Community Survey had 8k respondents and interesting results</li>
+  <li>Roadmap 2025 sets out plans for a new device database</li>
+</ul>
+</div>
+
+For all the juicy details, keep reading!<!--more-->
+
+## Open Home Foundation year one
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)